<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>629</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>QMTSim</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resource.qrc" >
    <normaloff>:/newPrefix/logo.jpg</normaloff>:/newPrefix/logo.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>33</y>
     <width>900</width>
     <height>577</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" colspan="12" >
     <widget class="QGraphicsView" name="view" >
      <property name="cursor" stdset="0" >
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QPushButton" name="addItemButton" >
      <property name="text" >
       <string>Add Polygon</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QPushButton" name="setPathButton" >
      <property name="text" >
       <string>Set Path</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" >
     <widget class="QPushButton" name="startAnimationButton" >
      <property name="text" >
       <string>Start Animation</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" >
     <widget class="QRadioButton" name="startAnimation_Radio" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="4" colspan="2" >
     <widget class="QProgressBar" name="animationProgressBar" >
      <property name="value" >
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="1" column="6" colspan="2" >
     <widget class="QSpinBox" name="animationtime" >
      <property name="statusTip" >
       <string>Time in milliseconds</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8" colspan="2" >
     <widget class="QPushButton" name="setAnimationTimeButton" >
      <property name="text" >
       <string>Set Animation Time</string>
      </property>
     </widget>
    </item>
    <item row="1" column="10" colspan="2" >
     <widget class="QSlider" name="animationSlider" >
      <property name="cursor" >
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QPushButton" name="addEllipseButton" >
      <property name="text" >
       <string>Add Ellipse</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="QPushButton" name="addCursorButton" >
      <property name="text" >
       <string>Add Cursor</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" >
     <widget class="QPushButton" name="deleteItemButton" >
      <property name="text" >
       <string>Delete Item</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2" >
     <widget class="QPushButton" name="saveProfileButton" >
      <property name="text" >
       <string>Save Item</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5" colspan="2" >
     <widget class="QPushButton" name="uploadItemButton" >
      <property name="text" >
       <string>Upload Item</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7" colspan="2" >
     <widget class="QPushButton" name="pushButton" >
      <property name="text" >
       <string>Save Session</string>
      </property>
     </widget>
    </item>
    <item row="2" column="9" colspan="2" >
     <widget class="QPushButton" name="pushButton_2" >
      <property name="text" >
       <string>Upload Session</string>
      </property>
     </widget>
    </item>
    <item row="2" column="11" >
     <spacer name="horizontalSpacer" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>547</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>610</y>
     <width>900</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>21</y>
     <width>900</width>
     <height>12</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>startAnimationButton</sender>
   <signal>clicked()</signal>
   <receiver>startAnimation_Radio</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>212</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel" >
     <x>272</x>
     <y>567</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
